_data/companies.yml

```
@@ -25,6 +25,13 @@
   visitors: Restricted 
   events: Restricted 
   last_update: 2020-03-10 
+  
+ - name: Alan
+  wfh: Required 
+  travel: Restricted 
+  visitors: Restricted 
+  events: Restricted 
+  last_update: 2020-03-12
```

```suggestion
   wfh: Required 
   travel: Restricted 
   visitors: Restricted 
   events: Restricted 
   last_update: 2020-03-12
```

Adding an extra space to keep formatting consistent